packages/components/package.json

```diff
@@ -10,32 +10,28 @@
   "homepage": "https://github.com/chanzuckerberg/edu-design-system/tree/main/packages/components",
   "license": "MIT",
   "main": "dist/index.js",
-  "publishConfig": {
-    "access": "public",
-    "directory": "dist/",
-    "registry": "https://registry.npmjs.org/"
-  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/chanzuckerberg/edu-design-system.git"
   },
+  "files": ["dist", "tailwind.config.js"],
```

Comment:
Since this is now publishing the whole directory as the package, this tells npm to only actually publish dist/ and tailwind.config.js.

npm will also include the package.json and README.md.